### PR TITLE
Update promise-polyfill to 7.0.2

### LINKED ISSUE #349: Promises broken in IE 11
### General information

* SDK version: 3.28+
* Browser and OS: IE

### Issue description

Version 7.0.0 of Promise Polyfill breaks Promises in IE as described here: https://github.com/taylorhakes/promise-polyfill/issues/67
This has been resolved in 7.0.1

### PR DESCRIPTION
**Pull request by Macavity**
| [Jira ticket]() |
|---|

### Summary

Fixes #349

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "braintree-web",
-  "version": "3.29.0",
+  "version": "3.28.1",
```
Comment (crookedneighbor) [inline package.json]: This should not be changed.

File: package.json [hunk 2]
```diff
@@ -21,10 +21,10 @@
     "@braintree/sanitize-url": "2.1.0",
     "@braintree/wrap-promise": "1.1.1",
     "card-validator": "4.2.0",
-    "credit-card-type": "6.1.1",
+    "credit-card-type": "6.1.0",
```
Comment (crookedneighbor) [inline package.json]: This should not be changed.

File: package.json [hunk 3]
```diff
@@ -48,7 +48,7 @@
     "gzip-size": "4.1.0",
     "is-uuid": "1.0.2",
     "jsdoc": "3.5.5",
-    "jsdoc-template": "braintree/jsdoc-template#3.2.0",
+    "jsdoc-template": "braintree/jsdoc-template#3.1.1",
```
Comment (crookedneighbor) [inline package.json]: This should not be changed.

**Comment by crookedneighbor** [comment]
Thanks for the report and the PR. Will test this out now and try to get a release out today.

**Comment by Macavity** [comment]
Thank you 👍

**Comment by crookedneighbor** [comment]
This definitely needs to be done, so I'm going to do the upgrade and release it, but so far I have not been able to reproduce the issue on the current version. Running Hosted Fields, for instance, in IE 9 and 11 works just fine for me whether I'm using the callback pattern or the promise pattern.

Can you give me steps to reproduce?

**Comment by Macavity** [comment]
We saw this error happening for our IE 11 users especially.

(TypeError) Das Objekt unterstützt diese Aktion nicht. (Rough translation, "The Object doesn't support this action) at Promise (function in webpack:///node_modules/braintree-web/client/get-configuration.js, Line 13

Line 13 is: 
`return new Promise(function (resolve, reject) {`

The stacktrace shows the call before was this one:
`return promiseOrCallback(fn.apply(this, args), callback);`

We had to revert back to 3.25.0 because we couldn't get my fork to work at all.

**Comment by crookedneighbor** [comment]
Hoping to get this out later today. Will update here when it is out.

**Comment by crookedneighbor** [comment]
This is out in v3.30.0

**Pull request merged**